{"name": "test_login_failure", "status": "passed", "start": 1732753717679, "stop": 1732753717840, "uuid": "872434ce-f111-47ef-8c05-9ea1e2b5b216", "historyId": "875475173a8d8fcaadc0586d31e090c2", "testCaseId": "875475173a8d8fcaadc0586d31e090c2", "fullName": "tests.test_login.TestLogin#test_login_failure", "labels": [{"name": "feature", "value": "Логин пользователя"}, {"name": "story", "value": "Неуспешный логин с неверными данными"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "admins-MacBook-Pro.local"}, {"name": "thread", "value": "23703-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login"}]}